src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Drawable;
+import core.basesyntax.model.Figure;
+import core.basesyntax.service.FigureSupplier;
+
+public class Main {
+    public static final int NUMBER_FIGURES = 12;
+    public static final int MAXIMUM_LENGTH = 100;
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[NUMBER_FIGURES];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < figures.length; i++) {
+            if (i % 2 == 0) {
```

Comment:
The first half of figures in this array should be generated with random parameters.